go/vt/vtgate/balancer/balancer.go

```diff
@@ -96,15 +97,35 @@ type TabletBalancer interface {
 	DebugHandler(w http.ResponseWriter, r *http.Request)
 }
 
-func NewTabletBalancer(localCell string, vtGateCells []string) TabletBalancer {
-	return &tabletBalancer{
+// NewTabletBalancer creates a new tablet balancer based on the specified mode.
+// Supported modes:
+//   - "flow": Flow-based balancer that maintains cell affinity while balancing load
```

Comment:
Flow is not very meaningful or descriptive to me. Can we rename this to something like `intra-cell`? What's the difference between this and `cell`?